TESTING.md

```diff
@@ -0,0 +1,34 @@
+# Running Tests
+
+## Prerequisites
+Before running tests, ensure all steps in the [BUILDING.md](https://github.com/apache/geode/blob/develop/BUILDING.md) have been performed.
+
+## Running Tests By Type
+Tests are broken up into three types - unit, integration, and distributed.  They can be executed using the following commands from the Geode repository root directory:
+* Unit tests
+  `./gradlew test --parallel`
+* Integration tests
+  `./gradlew integrationTest --parallel`
+* Distributed tests
+  `./gradlew distributedTest --parallel`
+
+## Running Individual Tests
+To run an individual test, you can either
+1. Run the test in your [IDE](https://github.com/apache/geode/blob/develop/BUILDING.md#setting-up-intellij)
+2. Run the from terminal by specifying the sub-project and test type:
+`./gradlew project:testType --tests testName`
```

Comment:
This needs to be formatted differently. The template and example commands display as a single run-on paragraph. Would be better to have each command displayed on a separate line. This is also true for the `Running Tests By Category` section

Reply:
Thanks Ken - will fix